proc_IR.py

```diff
@@ -33,40 +33,53 @@
             't2').argmax('t2').item()
     s.setaxis(t2-rough_center)
     #}}}
-    #{{{slicing out FID from echo and centering
-    s = slice_FID_from_echo(s,1,0) # so, to be clear, this includes all phasing?
+    #{{{hermitian function test and apply best shift
+    best_shift = hermitian_function_test(s[
+        'ph2',coh_sel['ph2']]['ph1',coh_sel['ph1']])
+    logger.info(strm("best shift is",best_shift))
+    s.ft('t2')
+    s *= exp(1j*2*pi*best_shift*s.fromaxis('t2'))
+    s.reorder(['ph2','ph1','indirect'])
+    s.ift('t2')
+    fl.next('time domain after hermitian test')
+    fl.image(s)
+    #}}}
+    #{{{zeroth order phase correction
+    ph0 = s['t2':0]['ph2',coh_sel['ph2']]['ph1',coh_sel['ph1']]
+    logger.info(strm(ndshape(ph0)))
+    if len(ph0.dimlabels) > 0:
+        assert len(ph0.dimlabels) == 1, repr(ndshape(ph0.dimlabels))+" has too many dimensions"
+        ph0 = zeroth_order_ph(ph0, fl=fl)
+        logger.info(strm('phasing dimension as one'))
+    else:
+        logger.info(strm("there is only one dimension left -- standard 1D zeroth order phasing"))
+        ph0 = ph0/abs(ph0)
+    s /= ph0
```

Comment:
so I was going to ask why add this back in -- you had removed it before, I looked at it, and thought at a glance this was all in `slice_FID_from_echo` --> on second look, the part at the end isn't, and I think belongs there.  I will make a subsequent commit.  My suggestion is that with that update, this re-addition here is no longer needed.  Please check.

Reply:
what re addition? this is outdated

Reply:
hmm, you just pushed those changes at some point over the last 12 minutes.

just to cross-reference, this pertains to [this comment](https://github.com/jmfrancklab/proc_scripts/pull/14#discussion_r442950285)

Reply:
okay so now I am confused, what happened here? you took this out and added it to the FID slice function?

Reply:
I was looking stuff over, and you apparently had stuff on your computer that you had not pushed, and you pushed them as I was writing these comments, hence the confusion.  I think these comments are still relevant to a comment I make below, which is why I'm leaving this open for now.

Reply:
OK -- looks like that other comment is dealt with